sakura_core/_main/CControlTray.cpp

```diff
@@ -810,7 +810,7 @@ LRESULT CControlTray::DispatchEvent(
 							// アイコンの登録
 							const CPlug::Array& plugs = CJackManager::getInstance()->GetPlugs( PP_COMMAND );
 							m_cMenuDrawer.m_pcIcons->ResetExtend();
-							for( CPlug::ArrayIter it = plugs.begin(); it != plugs.end(); it++ ) {
+							for( CPlug::ArrayIter it = plugs.cbegin(); it != plugs.cend(); it++ ) {
```

Comment:
```suggestion
							for( const auto it = plugs.cbegin(); it != plugs.cend(); it++ ) {
```
おそらく影響ないですが見やすくなる気がします。

Reply:
コンパイルエラーになります。
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2678	binary '++': no operator found which takes a left-hand operand of type 'const std::_Vector_const_iterator<std::_Vector_val<std::_Simple_types<_Ty>>>' (or there is no acceptable conversion)	sakura	D:\projects\sakura\sakura_core\_main\CControlTray.cpp	813	
```